[2023-01-07T05:09:07.614+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: bq_process.product_stats.execute_query manual__2023-01-07T05:08:48.661881+00:00 [queued]>
[2023-01-07T05:09:07.685+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: bq_process.product_stats.execute_query manual__2023-01-07T05:08:48.661881+00:00 [queued]>
[2023-01-07T05:09:07.686+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T05:09:07.688+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-07T05:09:07.690+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T05:09:07.786+0000] {taskinstance.py:1383} INFO - Executing <Task(BigQueryInsertJobOperator): product_stats.execute_query> on 2023-01-07 05:08:48.661881+00:00
[2023-01-07T05:09:07.804+0000] {standard_task_runner.py:55} INFO - Started process 339 to run task
[2023-01-07T05:09:07.826+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bq_process', 'product_stats.execute_query', 'manual__2023-01-07T05:08:48.661881+00:00', '--job-id', '542', '--raw', '--subdir', 'DAGS_FOLDER/bigquery_dag.py', '--cfg-path', '/tmp/tmpwo8evum8']
[2023-01-07T05:09:07.839+0000] {standard_task_runner.py:83} INFO - Job 542: Subtask product_stats.execute_query
[2023-01-07T05:09:08.197+0000] {task_command.py:376} INFO - Running <TaskInstance: bq_process.product_stats.execute_query manual__2023-01-07T05:08:48.661881+00:00 [running]> on host fdf4acf1d8ed
[2023-01-07T05:09:08.619+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bq_process
AIRFLOW_CTX_TASK_ID=product_stats.execute_query
AIRFLOW_CTX_EXECUTION_DATE=2023-01-07T05:08:48.661881+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-07T05:08:48.661881+00:00
[2023-01-07T05:09:08.623+0000] {connection.py:429} ERROR - Unable to retrieve connection from secrets backend (EnvironmentVariablesBackend). Checking subsequent secrets backend.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/connection.py", line 422, in get_connection_from_secrets
    conn = secrets_backend.get_connection(conn_id=conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/secrets/base_secrets.py", line 115, in get_connection
    return self.deserialize_connection(conn_id=conn_id, value=value)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/secrets/base_secrets.py", line 67, in deserialize_connection
    return Connection.from_json(conn_id=conn_id, value=value)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/connection.py", line 449, in from_json
    return Connection(conn_id=conn_id, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'key_path'
[2023-01-07T05:09:08.657+0000] {base.py:71} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-01-07T05:09:08.666+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-01-07T05:09:08.703+0000] {bigquery.py:1554} INFO - Inserting job ***_bq_process_product_stats_execute_query_2023_01_07T05_08_48_661881_00_00_e52a8595783ab969e02a3726ab25f9be
[2023-01-07T05:09:09.766+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1499, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1489, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 137, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Invalid schema update. Cannot add fields (field: page_view)

Location: asia-northeast3
Job ID: airflow_bq_process_product_stats_execute_query_2023_01_07T05_08_48_661881_00_00_e52a8595783ab969e02a3726ab25f9be

[2023-01-07T05:09:09.821+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=bq_process, task_id=product_stats.execute_query, execution_date=20230107T050848, start_date=20230107T050907, end_date=20230107T050909
[2023-01-07T05:09:09.883+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 542 for task product_stats.execute_query (400 Invalid schema update. Cannot add fields (field: page_view)

Location: asia-northeast3
Job ID: airflow_bq_process_product_stats_execute_query_2023_01_07T05_08_48_661881_00_00_e52a8595783ab969e02a3726ab25f9be
; 339)
[2023-01-07T05:09:09.926+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-01-07T05:09:10.118+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
